scripts/functional-tests.sh

```diff
@@ -41,16 +41,18 @@ cleanup() {
         "$TEST_CTX_SIZE_LAB_CHAT_LOG_FILE" \
         test_session_history
     rm -rf test_taxonomy
-    # revert port change from test_bind_port()
-    sed -i.bak 's/9999/8000/g' "${ILAB_CONFIG_FILE}"
 
     # revert model name change from test_model_print()
-    local original_model_name='merlinite-7b-lab-Q4_K_M'
-    local original_model_filename="${original_model_name}.gguf"
+    local original_model_filename="merlinite-7b-lab-Q4_K_M.gguf"
     local temporary_model_filename='foo.gguf'
-    sed -i.bak "s/baz/${original_model_name}/g" "${ILAB_CONFIG_FILE}"
     mv "${ILAB_CACHE_DIR}/models/${temporary_model_filename}" "${ILAB_CACHE_DIR}/models/${original_model_filename}" || true
-    rm -f "${ILAB_CONFIG_FILE}.bak" serve.log "${ILAB_CACHE_DIR}/models/${temporary_model_filename}"
+    rm -f serve.log "${ILAB_CACHE_DIR}/models/${temporary_model_filename}"
+
+    # reset config file and re-init defaults
```

Comment:
you could have a separate function for init (`function init()`) and then you could reuse it below in lines 609-616?)

Reply:
Added thanks.